db_upgrade_scripts/mosip_regprc/sql/1.2.0.1-B1_to_1.2.0.1_upgrade.sql

```diff
@@ -0,0 +1 @@
+\echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
```

Comment:
these should be 1.2.0.1-B2 to 1.2.0.1, since last release is B2 for regproc

Reply:
fixed